src/server/nserver.c

```diff
@@ -8850,6 +8850,19 @@ int Send_Guide(int Ind, byte search_type, int lineno, char* search_string) {
 	else return Packet_printf(&connp->c, "%c%c%d%s", PKT_GUIDE, search_type, lineno, search_string);
 }
 
+int Send_indicators(int Ind, u32b indicators) {
+	connection_t *connp = Conn[Players[Ind]->conn];
+	//player_type *p_ptr = Players[Ind];
+
+	if (!BIT(connp->state, CONN_PLAYING | CONN_READY)) {
+		errno = 0;
+		plog(format("Connection not ready for indicators (%d.%d.%d)",
+					Ind, connp->state, connp->id));
```

Comment:
I would make this a oneliner. Other lines above are longer, so it wouldn't stand out and wrapping can do an editor.